checkov/common/runners/object_runner.py

```diff
@@ -82,7 +96,8 @@ def run(
         for file_path in definitions.keys():
             self.pbar.set_additional_data({'Current File Scanned': os.path.relpath(file_path, root_folder)})
             skipped_checks = collect_suppressions_for_context(definitions_raw[file_path])
-            results = registry.scan(file_path, definitions[file_path], skipped_checks, runner_filter)  # type:ignore[arg-type]  # this is overridden in the subclass
+            results = registry.scan(file_path, definitions[file_path], skipped_checks,
+                                    runner_filter)    # this is overridden in the subclass
```

Comment:
`# type:ignore[arg-type]` not needed anymore? nice, then you can also remove the other comment 😄